From 7e04be09d1b0d161f14ccd73d62b68807f259ddf Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 23 Jan 2022 17:48:49 +0200
Subject: [PATCH] [SW-69581] gaudi: ppboot: increase sram footprint size

Preboot project code has been refactored and device specific headers
were removed from common code, this slightly increase the sram
footprint of gaudi ppboot application.
Adjusting the preboot text offset in the sram respectively.

Change-Id: Idb5a8ccfe1a45e2aabcf93e609046cfc15bf7bdd
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/board/habanalabs/gaudi/post-image-spi.sh b/board/habanalabs/gaudi/post-image-spi.sh
index 2061cec..03136cd 100755
--- a/board/habanalabs/gaudi/post-image-spi.sh
+++ b/board/habanalabs/gaudi/post-image-spi.sh
@@ -56,5 +56,5 @@
 
 PREBOOT_OFFSET=0x40000
 PREBOOT2_OFFSET=0x80000
-PREBOOT_TEXT_BASE=0x7FF0005000
+PREBOOT_TEXT_BASE=0x7FF0006000
 SHA_CHKSUM=sha256sum
